.travis.yml

```diff
@@ -7,6 +7,7 @@ notifications:
     email: false
 sudo: false
 addons:
+    ssh_known_hosts: github.com
```

Comment:
Can you add a comment explaining that this is needed for Xpress and why ?

Reply:
moved inside xpress